LR_Tourist/BLL/Interfaces/ITravelVoucher.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using BLL.Model;
+namespace BLL.Interfaces
+{
+    public interface ITravelVoucher
```

Comment:
ITravelVoucherService

Reply:
This service should be awaitable. Refactor all methods.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/

Reply:
fixed